### PR TITLE
archive controller 4.3 and add translations for 4.4

### PR DESCRIPTION
**Pull request by samccann**

### CODE REVIEW

File: data/controller_archive.yaml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Archive pages for Controller
+v_4_3:
```
Comment (oraNod) [inline data/controller_archive.yaml:2]:
```suggestion
v_4_3_0:
```

File: data/controller_i18n_links.yaml [hunk 2]
```diff
@@ -1,6 +1,14 @@
 # Translation links for Controller
 # Start Korean translations
 tower_ko:
+  v_4_4:
```
Comment (oraNod) [inline data/controller_i18n_links.yaml:4]:
```suggestion
  v_4_4_0:
```

File: data/controller_i18n_links.yaml [hunk 3]
```diff
@@ -35,6 +43,14 @@ tower_ko:
     upgrade: "http://docs.ansible.com/automation-controller/4.2.0/html_ko/upgrade-migration-guide/"
 # Start Japanese translations
 tower_ja:
+  v_4_4:
```
Comment (oraNod) [inline data/controller_i18n_links.yaml:46]:
```suggestion
  v_4_4_0:
```

File: data/controller_i18n_links.yaml [hunk 4]
```diff
@@ -327,6 +343,14 @@ tower_ja:
     upgrade: "http://docs.ansible.com/ansible-tower/3.1.4/html_ja/upgrade-migration-guide/"
 # Start Chinese translations
 tower_zh:
+  v_4_4:
```
Comment (oraNod) [inline data/controller_i18n_links.yaml:346]:
```suggestion
  v_4_4_0:
```

**Review by oraNod — CHANGES_REQUESTED** [review]
None of those 4.4 translation pages seem to be live. Is that OK?

**Comment by samccann** [comment]
Tested locally:
image

image
 etc etc

File: data/controller_archive.yaml (hunk 1 above)
Comment (samccann) [reply data/controller_archive.yaml:2]: Waiting for @tvo318 to confirm. When I follow the links, it isn't 4.3.0, it's 4.3 - https://docs.ansible.com/automation-controller/4.3/html/userguide/

Comment (tvo318) [reply data/controller_archive.yaml:2]: @oraNod right, I haven't published them yet.

Comment (tvo318) [reply data/controller_archive.yaml:2]: @oraNod is it ok to drop the .0 designation forever?

File: data/controller_i18n_links.yaml (hunk 2 above)
Comment (tvo318) [reply data/controller_i18n_links.yaml:4]: We are going to drop the _0 designation from now on out.

**Review by oraNod — APPROVED** [review]
Removed my suggestions after talking about the versions with @tvo318

**Review by tvo318 — APPROVED** [review]
Looks perfect, thanks @samccann! And thanks for the review and permission-add, @oraNod 🎉

**Comment by oraNod** [comment]
Thanks @samccann and @tvo318 I've merged and will create a PR for the docsite to get these changes in.

**Pull request merged**